src/lib/core/a11y/focus-trap.ts

```diff
@@ -11,48 +11,72 @@ import {coerceBooleanProperty} from '../coercion/boolean-property';
  * Things like tabIndex > 0, flex `order`, and shadow roots can cause to two to misalign.
  * This will be replaced with a more intelligent solution before the library is considered stable.
  */
-@Component({
-  moduleId: module.id,
-  selector: 'cdk-focus-trap, focus-trap',
-  templateUrl: 'focus-trap.html',
-  encapsulation: ViewEncapsulation.None,
+@Directive({
+  selector: 'cdk-focus-trap, focus-trap, [cdk-focus-trap], [focus-trap]',
```

Comment:
You don't need to add the un-prefixed attribute, we're trying to move to the prefixed versions. Also attribute should be [cdkFocusTrap]